common/configuration/asm.go

```diff
@@ -5,11 +5,10 @@ import (
 	"errors"
 	"fmt"
 	"net/url"
-	"sync"
 
 	"github.com/TBD54566975/ftl/internal/slices"
 
-	"github.com/alecthomas/types/optional"
+	. "github.com/alecthomas/types/optional" //nolint:stylecheck
```

Comment:
Not sure if ok @alecthomas ?

Reply:
Yeah I think for limited use this is fine, an optionals are a great use case.